pysteps/nowcasts/steps.py

```diff
@@ -33,6 +33,863 @@
     DASK_IMPORTED = False
 
 
+class StepsNowcaster:
+    def __init__(self, precip, velocity, timesteps, **kwargs):
+        # Store inputs and optional parameters
+        self.precip = precip
+        self.velocity = velocity
+        self.timesteps = timesteps
+        self.n_ens_members = kwargs.get("n_ens_members", 24)
+        self.n_cascade_levels = kwargs.get("n_cascade_levels", 6)
+        self.precip_thr = kwargs.get("precip_thr", None)
```

Comment:
```suggestion
        self.precip_threshold = kwargs.get("precip_threshold", None)
```
If this works for you, too, then we should change it in the rest of the code as well.

Reply:
What about vp_prep? I would change that one too but dont know what the prep stands for